views/index.hbs

```diff
@@ -1,97 +1,261 @@
+            <div data-o-grid-colspan="12">
+              <fieldset class="o-forms o-forms--wide">
+                <legend class="o-forms__label">Fit</legend>
+                <small class="o-forms__additional-info">
+                  Gets images as masks, squashed or solid colours - <i>(fill default)</i>
+                </small>
+                <div class="o-forms__group o-forms__group--inline">
+                  <input type="radio" id="fit-fill" name="fit" value="fill" class="o-forms__radio" checked="">
+                  <label for="fit-fill" class="o-forms__label">fill</label>
+                  <input type="radio" id="fit-cover" name="fit" value="cover" class="o-forms__radio">
+                  <label for="fit-cover" class="o-forms__label">cover</label>
+                  <input type="radio" id="fit-solid" name="fit" value="solid" class="o-forms__radio">
+                  <label for="fit-solid" class="o-forms__label">solid</label>
+                </div>
+              </fieldset>
+            </div>
+          </div>
+          <div class="o-grid-row o-grid-row--compact">
+            <div data-o-grid-colspan="12">
+              <fieldset class="o-forms o-forms--wide">
+                <legend class="o-forms__label">Order</legend>
+                <small class="o-forms__additional-info">
+                  Display order of images - <i>(colour default)</i>
+                </small>
+                <div class="o-forms__group o-forms__group--inline">
+                  <input type="radio" id="order-published" name="order" value="published" class="o-forms__radio" checked="">
+                  <label for="order-published" class="o-forms__label">published</label>
+                  <input type="radio" id="order-colour" name="order" value="colour" class="o-forms__radio">
+                  <label for="order-colour" class="o-forms__label">colour</label>
+                </div>
+              </fieldset>
+            </div>
+          </div>
+          <div class="o-grid-row o-grid-row--compact">
+            <div data-o-grid-colspan="12">
+              <fieldset class="o-forms o-forms--wide">
+                <legend class="o-forms__label">Sort</legend>
+                <small class="o-forms__additional-info">
+                  Sort order by asc or desc - <i>(asc default)</i>
+                </small>
+                <div class="o-forms__group o-forms__group--inline">
+                  <input type="radio" id="sort-asc" name="sort" value="asc" class="o-forms__radio" checked="">
+                  <label for="sort-asc" class="o-forms__label">ascending</label>
+                  <input type="radio" id="sort-desc" name="sort" value="desc" class="o-forms__radio">
+                  <label for="sort-desc" class="o-forms__label">descending</label>
+                </div>
+              </fieldset>
+            </div>
+          </div>
+        </div>
+
+        
+
+        
+
+        <!--
```

Comment:
Please don't leave big commented out chunks of code

Reply:
Fixed!